UnityProject/Assets/Scripts/Systems/Explosions/NodeTypes/ExplosionNode.cs

```diff
@@ -127,6 +110,33 @@ public virtual float DoDamage(Matrix matrix, float DamageDealt, Vector3Int v3int
 			return EnergyExpended;
 		}
 
+		protected void DamageLayers(float DamageDealt, Vector3Int v3int)
+		{
+			if (DamageDealt < 100) return;
+			var Node = matrix.GetMetaDataNode(v3int);
+			if (Node == null) return;
+			foreach (var electricalData in Node.ElectricalData)
+			{
+				electricalData.InData.DestroyThisPlease();
+			}
+			if (DamageDealt > 135)
+			{
+				foreach (var disposalPipe in Node.DisposalPipeData)
+				{
+					matrix.TileChangeManager.MetaTileMap.RemoveTileWithlayer(disposalPipe.NodeLocation, LayerType.Disposals);
```

Comment:
Any future plans to spawn broken pipe ends / detached pipe objects?

Reply:
does it spawn pipes in ss13? Because I don't recall any items getting dropped when bombs tear through the station.